xpanse/api/incident_management/v1/alerts.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, List, Optional
+
+from xpanse.const import V1_PREFIX
+from xpanse.endpoint import XpanseEndpoint
+from xpanse.response import XpanseResponse
+from xpanse.utils import build_request_payload
+
+
+class AlertsEndpointV1(XpanseEndpoint):
+    """
+    Part of the Public API for handling Alert Updates Endpoint.
+    See: https://docs-cortex.paloaltonetworks.com/r/Cortex-Xpanse-REST-API/Update-Alerts
+    """
+
+    UPDATE_ENDPOINT = f"{V1_PREFIX}/alerts/update_alerts"
+    UPDATE_DATA_KEY = "alerts_ids"
+
+    def update(
+        self, alert_id_list: List[str], update_data: Any, **kwargs: Any
```

Comment:
I think currently we also allow the user to pass in a custom `request_data` parameter here. According to the docs, the `requst_data` only accepts `alert_id_list` and `update_data` anyway - so I think we can use discretion here

Reply:
Good point, I'll go ahead and remove the **kwargs